docs/source/getting-started.mdx

```diff
@@ -346,6 +346,12 @@ of schemas, resolvers, and generating types:
 - [Resolvers](./data/resolvers/)
 - [Generating TS types for your schema](./workflow/generate-types)
 
-Want to learn how to modularize and scale a GraphQL API? Check out the [Apollo Federation Docs](/federation) to learn how a federated architecture can create a unified _supergraph_ that combines multiple GraphQL APIs.
```

Comment:
I removed this since we're moving away from a federation-first mindset and don't want to crowd the Next Steps section with too many avenues.